script/build

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+'use strict';
+
+let rollup = require('rollup').rollup;
+let babel = require('rollup-plugin-babel');
+let babelrc = require('babelrc-rollup').default;
+let filesize = require('file-size');
+let gzipSize = require('gzip-size').sync;
+let stat = require('fs').statSync;
+let minify = require('uglify-js').minify;
+let pkg = require('../package.json');
+
+rollup({
```

Comment:
> The minifier I used doesn't work with ES6 (uglify-js).

I might have a [solution](http://prosecco.surge.sh/) to that problem... hoping to open source it in a few days